.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
In rpm-ostree at least, we do `weekly` to have some sort of batching at the notifications level. Though here we don't really have that many deps so many it's fine.

Reply:
That's a good point.  We're doing `daily` in the Rust projects and the paperwork load is honestly a bit high.  I'll PR to update everything.